lib/CMakeLists.txt

```diff
@@ -18,10 +18,15 @@ set(PROJECT_SOURCES
     src/values/abstract_config_value.cc
     src/values/config_boolean.cc
     src/values/config_number.cc
-    src/values/config_int.cc
```

Comment:
One reason for my original decision was that the Java's equality testing for config_nubmers deliberately ignores the distinction. This means that the tests I wrote still pass, despite the fact I created config_longs everywhere, and they all should be ints now. I guess this isn't a problem?